getting-started/getting-started.md

```diff
@@ -245,43 +300,58 @@ After applying these steps, the relevant portions of your build file will look l
 <build>
 ```
 {% endcode %}
-{% endtab %}
 
-{% tab title="Gradle" %}
+To double-check that everything is working, run the command `./mvnw rewrite:run`. Your project should be upgraded to Spring Boot 2 and all of the test classes should be updated to JUnit 5. Your `pom.xml` file will also have had its Spring dependencies updated, the JUnit 4 dependency removed, and the JUnit 5 dependency added.
+
+### Gradle + External Modules
+
+Unlike Maven projects, Gradle projects have two options for specifying versions for recipes. You can:
+
+1. Add `rewrite-recipe-bom` as a dependency
+2. Add `rewrite-spring` as a dependency with a version specified
+
+If you choose to use the `rewrite-recipe-bom`, you won't have to worry about specifying versions for your rewrite recipes as all of the recipes you include in your `dependencies` section will have an appropriate version specified in the bill of materials (BOM). For Gradle projects, this is the recommended approach.
+
+If you choose to not use the `rewrite-recipe-bom`, you'll need to ensure that you have the appropriate version of all of your dependencies included (as you would with Maven projects).
```

Comment:
```suggestion
If you choose to not use `rewrite-recipe-bom`, you'll need to specify the version of each rewrite recipe module you use. 
```

Reply:
That's a great call out. Agree with this suggestion. Thanks